Cargo.toml

```diff
@@ -1,3 +1,5 @@
+cargo-features = ["profile-rustflags"]
```

Comment:
if add this, we can't use stable Rust anymore. And this is seems no need anymore, so let's remove this line